src/endorsement/endorsement.service.ts

```
@@ -11,6 +11,9 @@ import { NotificationsService } from '../notifications/notifications.service';
 
 @Injectable()
 export class EndorsementService {
+  getUserEndorsements(userId: string): any {
```

ESLint

[error] 'userId' is defined but never used.

(@typescript-eslint/no-unused-vars)